package.json

```diff
@@ -18,7 +18,7 @@
     "mocha-steps": "^1.3.0",
     "prettier": "1.19.1",
     "ts-node": "^10.9.1",
-    "typescript": "^4.9.4",
+    "typescript": "^4.9.5",
```

Comment:
The `package-lock.json` file also needs to be checked in.

Reply:
Will do